frontend/src/FullPageRoutes/VideoListPage.tsx

```diff
@@ -1,33 +1,34 @@
-import * as React from 'react';
-import { request } from '../utils/fetch';
-
-type Video = {
-    id: number;
-    title: string;
-    video_file: string;
-};
-
-export default function VideoListPage() {
-    const [videos, setVideos] = React.useState<Video[]>([]);
-
-    React.useEffect(() => {
-        request<Video[]>('/api/video/', 'get', true, false).then((res) => {
-            setVideos(res.parsedBody);
-        });
-    });
-
-    return (
-        <ul>
-            {videos.map(({ id, title, video_file }) => {
-                return (
-                    <li key={'video' + id}>
-                        {title}
-                        <video width={500} height={500} controls>
-                            <source type='video/mp4' src={video_file} />
-                        </video>
-                    </li>
-                );
-            })}
-        </ul>
-    );
-}
+import * as React from "react"
+import { request } from "../utils/fetch"
+
+type Video = {
+  id: number
+  title: string
+  video_file: string
+}
+
+export default function VideoListPage(): JSX.Element {
+  const [videos, setVideos] = React.useState<Video[]>([])
+
+  React.useEffect(() => {
+    request<Video[]>("/api/video/", "get", true, false).then((res) => {
```

Comment:
Check your arguments here; I don't think you are intending to specify false as the body of the request.